src/js/disability-benefits/components/ClaimDetailLayout.jsx

```diff
@@ -15,13 +15,14 @@ export default class ClaimDetailLayout extends React.Component {
 
     let content;
     if (!loading) {
+      const claimsLink = `your-claims${claim.attributes.open ? '' : '/closed'}`;
```

Comment:
There were a few breadcrumbs I didn't update as they shouldn't be reachable for a closed claim and didn't contain the full claim object